includes/fbproductfeed.php

```diff
@@ -89,6 +89,10 @@ public function schedule_feed_generation() {
 		 * @since 1.11.0
 		 */
 		public function generate_feed() {
+			$profiling_logger = facebook_for_woocommerce()->get_profiling_logger();
+			$profiling_logger->start( 'generate_feed' );
+
+			do_action( 'qm/start', 'facebook_for_woocommerce__generate_feed' );
```

Comment:
There's also some other timing code in this function - see `set_feed_generation_time_with_decay`  on line 107. We should consider combining these into one, and/or see if that feed generation time data is still being used.